Answers.md

```diff
@@ -1 +1,23 @@
 
+1. Describe the biggest difference between `.forEach` & `.map`.
+.forEach doesn't return anything. It just calls a callback function for each item in an array. .map returns a new array.
+
+2. What is the difference between a function and a method?
+A method needs an object whereas a function does not.
+
+3. What is closure?
```

Comment:
Good example! This is a good way to show your knowledge. You should consider practicing making examples and explaining your work. It really helps you later on.